### PR TITLE
Feature/graph layout

### LINKED ISSUE #118: Improve Graph Rendering
Our graph layout algorithm currently uses [d3-hierarchy](https://github.com/d3/d3-hierarchy) to position vertices and edges. The object that is used to construct this layout is a JSON object with a single root node. The Marquez API outputs DAGs which often do not have the property. Therefore, we are limited in our ability to render a layout that reflects the full knowledge of the Marquez data model. We plan on sunsetting the use of `d3-hierarchy` and use [dagre](https://github.com/dagrejs/dagre) in its place, which allows us to render real DAGs and configure layout flows.

In addition to improving our layout positioning, we also plan on incorporating Airbnb's excellent visualization library [visx](https://github.com/airbnb/visx) to assist with zooming, edge layouts, and enable future iterations on supplemental graph interactions.

There are many tickets in the backlog that allude to this issue and we hope to rectify the shortcomings with our upcoming changes.

### PR DESCRIPTION
**Pull request by phixMe**
Fixes #118 

### Description
This replaces the entire graph rendering algorithm with `dagre.js` as outlined in the linked issue.

### Preview
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,22 +17,32 @@
     "node": "10.16.3"
   },
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.12",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
```
Comment (julienledem) [inline package.json:22]: typo?
fort/font

**Review by julienledem — APPROVED** [review]
this looks good to me!

File: package.json (hunk 1 above)
Comment (phixMe) [reply package.json:22]: @julienledem It seems that this is correct. The organization that ships `fontAwesome` is called `fortAwesome`. It's a bit confusing, but here is their GitHub organization page: 
https://github.com/FortAwesome

**Review by wslulciuc — APPROVED** [review]
This is AMAZING! 💯 🥇 🚀

**Pull request merged**